api/src/custom.pipe.ts

```diff
@@ -58,6 +58,16 @@ export class FilterUserPipe implements PipeTransform {
             },
           };
         }
+
+        if (filters.id) {
+          return {
```

Comment:
At the moment we only allow one filter at a time. 
This will be changed if we have time